Core/Libraries/Source/WWVegas/WW3D2/decalsys.cpp

```diff
@@ -47,6 +47,7 @@
  *   DecalGeneratorClass::Set_Mesh_Transform -- sets the current mesh coordinate system        *
  * - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - */
 
+#include <Utility/CppMacros.h>
```

Comment:
These includes should not be necessary in most cases. The include should come through WWCommon already. If not, then an always.h (WWVegas) or PreRTS.h (GameEngine) include is missing somewhere. GameEngineDevice does not use PreRTS.h there is may be necessary sometimes, but I would expect not in many cases because WWVegas stuff gets always.h